apps/sms/views/shared/js/navigation.js

```diff
@@ -583,24 +583,36 @@ function switchPanel() {
   var isGoingBack = oldView && oldView.previous === newView.name;
 
   newPanelElement.style = '';
+  newPanelElement.classList.remove('panel-hidden');
+  newPanelElement.classList.add('panel-active');
```

Comment:
I checked with my workload that with a current gecko this does not provoke big reflows anymore. But if you can double check with a big workload also it would be nice.